### ISSUE TITLE
Events triggered in strange ways

### ISSUE THREAD

**Issue by strendey** [kwhat/jnativehook#374]
Multiple events triggered by pressing the key only once.
The first few times the key is pressed everything works normal, but after a few keys are pressed this behavior occurs.
![image](https://user-images.githubusercontent.com/68207170/138008391-440279e7-f6f8-4f7b-967a-ae9ab98fcabc.png)
I only pressed each key once, as you can see from the IntelliJ console, but the event was fired several times.
Code: 
![image](https://user-images.githubusercontent.com/68207170/138007984-7598a1ab-de35-4c4c-bdba-045c21d8da55.png)
It's a simple code, I've used it several times, only this time it's causing problems.

**Comment by kwhat** [comment]
Operating system?

**Comment by kwhat** [comment]
Hmm, I just looked at your code.  Try this:

```
public void nativeKeyPressed(NativeKeyEvent event) {
    // If you want the AWT key name, use this.  
    // getRawCode is used to get the native keycode which isn't what you want for this function or almost anything else.  It is really only good for binding a system specific key or debugging.
    System.out.println(KeyEvent.getKeyText(event.getKeyCode()));
}
```

```
public void nativeKeyTyped(NativeKeyEvent event) {
    // If you are looking for the text that a key press produced, you want this.
    // This will only be useful on key typed events.
    System.out.println(event.getKeyChar());
}
```

**Comment by strendey** [comment]
> Sistema operacional?

Windows 10

> Hmm, acabei de olhar seu código. Experimente isto:
> 
> ```
> public void nativeKeyPressed(NativeKeyEvent event) {
>     // If you want the AWT key name, use this.  
>     // getRawCode is used to get the native keycode which isn't what you want for this function or almost anything else.  It is really only good for binding a system specific key or debugging.
>     System.out.println(KeyEvent.getKeyText(event.getKeyCode()));
> }
> ```
> 
> ```
> public void nativeKeyTyped(NativeKeyEvent event) {
>     // If you are looking for the text that a key press produced, you want this.
>     // This will only be useful on key typed events.
>     System.out.println(event.getKeyChar());
> }
> ```

Thank you very much! I had tried this method to get the key in nativeKeyPressed and it didn't work, now I understand that it only works with keyTyped. If this helps: I recorded a video where it shows this spam occurring without me pressing any key, I just went to create a new issue on GitHub and when I came back the console was spamming madly nonstop (without me pressing any key).

**Comment by kwhat** [comment]
I doubt I will be able to reproduce the key bounce you are seeing.  Try something like this before you register the hook and lets see if you are getting excessive messages like "Restarting Windows input hook on window event:" or something else interesting.

```
// Get the "com.github.kwhat.jnativehook" logger and set the level
Logger logger = Logger.getLogger(GlobalScreen.class.getPackage().getName());
logger.setLevel(Level.ALL);

// Don't forget to disable the parent handlers.
logger.setUseParentHandlers(false);
```

They hook only fires key events when the OS reports them so for some reason the OS is probably getting these events.  I will run the hook for a little while tomorrow to see if it shows up for me.

**Comment by strendey** [comment]
> I doubt I will be able to reproduce the key bounce you are seeing. Try something like this before you register the hook and lets see if you are getting excessive messages like "Restarting Windows input hook on window event:" or something else interesting.
> 
> ```
> // Get the "com.github.kwhat.jnativehook" logger and set the level
> Logger logger = Logger.getLogger(GlobalScreen.class.getPackage().getName());
> logger.setLevel(Level.ALL);
> 
> // Don't forget to disable the parent handlers.
> logger.setUseParentHandlers(false);
> ```
> 
> They hook only fires key events when the OS reports them so for some reason the OS is probably getting these events. I will run the hook for a little while tomorrow to see if it shows up for me.

Sorry for the long answer, I was only able to log into the computer today.
For some reason, everything was working normally today.
I did what you asked, everything went normally, without any failures or messages that were not in my code.
